Source/DafnyLanguageServer.Test/Diagnostics/PublishedVerificationStatus.cs

```diff
@@ -0,0 +1,24 @@
+using Microsoft.Dafny.LanguageServer.Workspace;
+using Xunit;
+
+namespace Microsoft.Dafny.LanguageServer.IntegrationTest.Synchronization; 
+
+public class PublishedVerificationStatusTests {
+  [Fact]
+  public void CombineTest() {
+    Assert.Equal(PublishedVerificationStatus.Running,
+      NotificationPublisher.Combine(PublishedVerificationStatus.Stale, PublishedVerificationStatus.Running));
+    Assert.Equal(PublishedVerificationStatus.Running,
+      NotificationPublisher.Combine(PublishedVerificationStatus.Queued, PublishedVerificationStatus.Running));
+    Assert.Equal(PublishedVerificationStatus.Queued,
+      NotificationPublisher.Combine(PublishedVerificationStatus.Stale, PublishedVerificationStatus.Queued));
+    Assert.Equal(PublishedVerificationStatus.Stale,
+      NotificationPublisher.Combine(PublishedVerificationStatus.Stale, PublishedVerificationStatus.Error));
+    Assert.Equal(PublishedVerificationStatus.Running,
```

Comment:
How much did you weight this one against `PublishedVerificationStatus.Error`? I understand why it's running, but for example in the gutter icons, errors prevail above everything, including running.

Reply:
I considered it. I think both have downsides:
- If you show running, then someone who's only watching the status bar to see whether errors are found, will wait longer than they have too. 
- If you show error, then someone might be surprised when new errors appear after it already went from running to error.